pallets/gear-program/src/add_section_sizes_migration.rs

```diff
@@ -34,22 +34,20 @@ use {
     sp_std::vec::Vec,
 };
 
-const MIGRATE_FROM_VERSION: u16 = 3;
-const MIGRATE_TO_VERSION: u16 = 4;
-const ALLOWED_CURRENT_STORAGE_VERSION: u16 = 5;
+const MIGRATE_FROM_VERSION: u16 = 5;
+const MIGRATE_TO_VERSION: u16 = 6;
+const ALLOWED_CURRENT_STORAGE_VERSION: u16 = 6;
 
-pub struct AppendStackEndMigration<T: Config>(PhantomData<T>);
+pub struct AddSectionSizesMigration<T: Config>(PhantomData<T>);
```

Comment:
please implement separated migration

Reply:
I implemented it as a separate migration in `pallets/gear-program/src/add_section_sizes_migration.rs` and removed `AppendStackEndMigration` because it seems to have already been applied to the Vara chain.

 Do we still need the `AppendStackEndMigration` migration?

Reply:
It seems like it will be a better idea to move the specific migration to a subdirectory, as done in #3963.

Reply:
yeah, that's exactly I was talking about. fetch master pls

Reply:
Fixed